### PR TITLE
Upp inline post for RRFS and HAFS

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
## Description

This PR will update inline post with changes:

- Output lat/lon of four corner points in text file for RRFS rotated lat-lon gird
- Read instant model top longwave from model
- Update upp submodule revision with the recent commits from UPP repository.

Is a change of answers expected from this PR?  No.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues should always be created before starting work on a PR branch!)
- fixes #[1359](https://github.com/ufs-community/ufs-weather-model/issues/1359)

## Testing

How were these changes tested?  
What compilers / HPCs was it tested with?  
Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)  Yes.
Have the ufs-weather-model regression test been run? On what platform?  Yes, on WCOSS2 dogwood.
- Will the code updates change regression test baseline? If yes, why? Please show the baseline directory below.
   No changes in regression test baseline right now because the model is not set to output instant model top longwave yet.
- Please commit the regression test log files in your ufs-weather-model branch

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
Those branches should have matching names (ideally)

Do PRs in upstream repositories need to be merged first? No.

### CODE REVIEW

**Comment by jkbk2004** [comment]
Upstream ufs-wm [PR#1360](https://github.com/ufs-community/ufs-weather-model/pull/1360) was fully tested ok. Can we have final review and start merging in the PR?

File: io/post_fv3.F90 [hunk 1]
```diff
@@ -3265,6 +3286,18 @@ subroutine set_postvars_fv3(wrt_int_state,mpicomp,setvar_atmfile,   &
 !
 !more fields need to be computed
 !
+
+! write lat/lon of the four corner point for rotated lat-lon grid
+      if(mype == 0 .and. maptype == 207)then
+        write(flatlon,1001)ifhr
+        open(112,file=trim(flatlon),form='formatted',status='unknown')
```
Comment (junwang-noaa) [inline io/post_fv3.F90:3293]: @WenMeng-NOAA you may want to check if a unit number(112) is available, this can be done in the future though.

**Review by junwang-noaa — APPROVED** [review]

File: io/post_fv3.F90 (hunk 1 above)
Comment (WenMeng-NOAA) [reply io/post_fv3.F90:3293]: @junwang-noaa Do you have a sample for inquiry of an unit number? Thanks!

**Review by DeniseWorthen — APPROVED** [review]

File: io/post_fv3.F90 (hunk 1 above)
Comment (DeniseWorthen) [reply io/post_fv3.F90:3293]: I think you could also use newunit:

```
     open (newunit=iunit, trim(flatlon),form='formatted',status='unknown')
     write(iunit,1002).....
     close(iunit)
```

Comment (WenMeng-NOAA) [reply io/post_fv3.F90:3293]: @DeniseWorthen Thanks for the solution. I will add it in my next PR.

**Pull request merged**